src/cryptoadvance/specter/devices/bitcoin_core.py

```diff
@@ -1,10 +1,13 @@
 import os, shutil
-from bip32 import BIP32
 from mnemonic import Mnemonic
 from hwilib.descriptor import AddChecksum
 from ..device import Device
-from ..helpers import alias, convert_xpub_prefix, encode_base58_checksum, get_xpub_fingerprint, seed_to_hd_master_key
+from ..helpers import (alias, convert_xpub_prefix,
+                       encode_base58_checksum, decode_base58,
+                       get_xpub_fingerprint)
 from ..key import Key
+from io import BytesIO
+import hmac
```

Comment:
Seems like you are missing import `hashlib` and `b58encode` from helpers... But after adding them like so and retrying it works great :)
```suggestion
import hmac
import hashlib
from ..helpers import b58encode
```

Reply:
I reused `encode_base58_checksum` instead and also deleted `b58encode` function that is not used anymore.